packages/coding-standards/tests/Unit/CsFixer/Phpdoc/NoUselessAccessFixer/NoUselessAccessFixerTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\CodingStandards\CsFixer\OrmJoinColumnRequireNullableFixer;
+
+use Symplify\EasyCodingStandardTester\Testing\AbstractCheckerTestCase;
+
+final class NoUselessAccessFixerTest extends AbstractCheckerTestCase
```

Comment:
The name evokes in me that this fixer will also fix (as the access annotation is useless)
```php
class ...
/**
 * @access
 */
const X = 10;
```

But it won't... Am I right? If not maybe think about a better name

Reply:
If you decide that this fixer will also remove such empty `@access`, I'll be happy and you don't have to think about renaming

Reply:
Empty access annotations are now removed too